webpack-plugins/webpack-less-plugin.js

```diff
@@ -4,6 +4,26 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const PostCssFlexBugFixes = require('postcss-flexbugs-fixes');
 const postcssLoadConfig = require('postcss-load-config');
 
+const interpolateName = require('loader-utils').interpolateName;
+
+function normalizePath(file) {
+  return path.sep === '\\' ? file.replace(/\\/g, '/') : file;
+}
+
+// Custom function to not use 'loaderContext._module.matchResource' in hashing CSS class name.
+function getLocalIdent(loaderContext, localIdentName, localName, options) {
+  let relativeMatchResource = '';
+
+  const relativeResourcePath = normalizePath(
+    path.relative(options.context, loaderContext.resourcePath),
+  );
+
+  // eslint-disable-next-line no-param-reassign
+  options.content = `${options.hashPrefix}${relativeMatchResource}${relativeResourcePath}\x00${localName}`;
```

Comment:
I don't see a compelling reason to interpolate the empty string

```suggestion
  const relativeResourcePath = normalizePath(
    path.relative(options.context, loaderContext.resourcePath),
  );

  // eslint-disable-next-line no-param-reassign
  options.content = `${options.hashPrefix}${relativeResourcePath}\x00${localName}`;
```

Reply:
@wesleybl please take a look at David's suggestion.

Reply:
@davisagli @sneridagh I had left it as it was, to be as similar as possible to the original code, in order to facilitate future comparisons. But I don't know if that makes a lot of sense. So, I made the changes suggested by David.